src/build.py

```diff
@@ -633,7 +634,7 @@ def SyncRepos(filter=None, sync_lkgr=False):
   buildbot.Step('Sync Repos')
 
   good_hashes = None
-  if options.sync_lkgr:
```

Comment:
Huh. This technically works but is gross, right? Wouldn't have expected it to work, learned something today about Python scope resolution.

Thanks!